src/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -194,6 +196,45 @@ public static bool IsReferenceOrContainsReferences<T>()
             // See getILIntrinsicImplementation for how this happens.
             throw new InvalidOperationException();
         }
+
+        // Returns true iff the object has a component size;
```

Comment:
nit: typos `iif`

Reply:
No, it's correct here -> https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Ah ok, never mind! thank's @gfoidl !